translations/CONTRIBUTING.hindi.md

```diff
@@ -0,0 +1,151 @@
+
+### सामुदायिक जानकारी
+
+जबसे हमने अपना [सीखना पृष्ठ](https://hasura.io/learn/) लॉन्च किया, हम योगदानों से ख़ुशि हैं:
+
+- मौजूदा ट्यूटोरियल में गलतियों को ठीक करें
+
+- नए ट्यूटोरियल जोड़ें (कृपया हमें पहले आपके विचार बताएं ताकि डुप्लीकेट ना हो)
+
+जानने की रिपॉजिटरी का README [यहां](https://github.com/hasura/learn-graphql) पाया जा सकता है।
+
+सीखने की जानकारी के अलावा, तकनीकी सामुदायिक जानकारी में योगदान करने के तीन अन्य तरीके हैं:
+
+- [बॉयलरप्लेट्स](https://github.com/hasura/graphql-engine/tree/master/community/boilerplates)
+
+- [नमूना ऐप्स](https://github.com/hasura/graphql-engine/tree/master/community/sample-apps)
+
+- [उपकरण](community/tools)
+
+यदि आप समुदाय को योगदान देना चाहते हैं निम्नलिखित माध्यमों से, तो हमारी [समुदाय विकि](https://github.com/hasura/graphql-engine/wiki/Community-Wiki) देखें।:
+
+- टेक्निकल ब्लॉगपोस्ट लिखना
+
+- किसी इवेंट में बोलना
+
+- एक कार्यशाला का आयोजन
+
+यदि आप ऊपर बताई गई चीज़ की अलावा कुछ और जोड़ना चाहते हैं, एक पुल रिक्वेस्ट ज़रूर भेजें।
+
+### हसुरा सीएलआई
+
+हमारे पास सीएलआई पर कुछ मुद्दे हैं जो ओपन सोर्स के योगदान के लिए उपयुक्त हैं। यदि आप गो जानते हैं या आप इस पर काम करके सीखना चाहते हैं, तो इन [मुद्दों](https://github.com/hasura/graphql-engine/issues?q=is%3Aopen+is%3Aissue+label%3Ac%2Fcli+label%3A%22help+wanted%22) को देखें।
+
+CLI रिपॉजिटरी की README को [यहाँ](https://github.com/hasura/graphql-engine/tree/master/cli) पाया जा सकता है।
+
+<a name="ways"> </a>
+
+## 5. योगदान के तरीके
+
+### किसी समस्या की रिपोर्ट करना
+
+- सुनिश्चित करें कि आपने नवीनतम संस्करण के खिलाफ परीक्षण किया हैं। यह संभव है कि हम आपके द्वारा अनुभव किए जा रहे बग को पहले ही सही कर लिए हैं।
+
+- समस्या को पुन: उत्पन्न करने के लिए कदम प्रदान करें, साथ ही पोस्टग्रेज संस्करण, ग्राफ्क्यूएल-इंजन संस्करण और प्रदाता(हरोकू, डॉकर, आदि) जिसे आप चला रहे हैं।
+
+- यदि आवश्यक हो तो कृपया सर्वर के लॉग भी शामिल करें।
+
+### किसी मुद्दे पर काम करना
+
+- हम [फोर्क-और-ब्रांच गिट वर्कफ़्लो](https://blog.scottlowe.org/2015/01/27/use-fork-branch-git-workflow/) का उपयोग करते हैं।
+
+- कृपया सुनिश्चित करें कि आप जो काम कर रहे हैं, उससे जुड़ा एक इश्यू है।
+
+- यदि आप किसी मुद्दे पर काम कर रहे हैं, तो कृपया टिप्पणी करें ताकि दूसरों के द्वारा डुप्लिकेट काम ना हो।
+
+- अपने कमिट को स्क्वाश करें और अंत में कमिट मैसेज में `फिक्स #<इश्यू-नो>` या `क्लोज़ #<इश्यू-नो>` का इस्तेमाल करें।
+
+    उदाहरण: `हर चीज़ का उत्तर हल करें (फिक्स #42)` या `हर चीज़ का उत्तर हल करें, फिक्स #42`
+
+- पुल रिक्वेस्ट सबमिट करने से पहले अपनी ब्रांच के साथ मास्टर को रीबेस करें।
+
+<a name="commit-messages"></a>
+
+## 6. कमिट संदेश
+
+- पहली पंक्ति में परिवर्तनों का सारांश होना चाहिए, 50 से अधिक अक्षर नहीं, इसके बाद एक वैकल्पिक निकाय जिसमे परिवर्तनों के बारे में अधिक विवरण हो। अच्छा कमिट सन्देश लिखने के बारे में अधिक जानकारी के लिए [इस लिंक](https://github.com/erlang/otp/wiki/writing-good-commit-messages) का संदर्भ लें। 
+
+- अत्यावश्यक वर्तमान काल का उपयोग करें: "ऐड/फिक्स/चेंज", न कि "एडेड/फिक्स्ड/चेंज्ड" और "एड्स/फिक्सेस/चेंजेस"।
```

Comment:
Please reframe the sentence to use English counterparts.

अत्यावश्यक वर्तमान काल का उपयोग करें : "add/fix/change" , न कि "added/fixed/changed"  या  "adds/fixes/changes"।